lib/Server.js

```diff
@@ -3254,9 +3254,36 @@ class Server {
    * @param {string | string[]} watchPath
    * @param {WatchOptions} [watchOptions]
    */
-  watchFiles(watchPath, watchOptions) {
+  watchFiles(watchPath, watchOptions = {}) {
     const chokidar = require("chokidar");
-    const watcher = chokidar.watch(watchPath, watchOptions);
+    const isGlob = require("is-glob");
```

Comment:
Why do not use https://github.com/webpack/webpack-dev-server/issues/5368#issuecomment-2520984933 ?

Reply:
See my other comment.

is-glob is also part of parent-glob so it's not actually pulling in more dependencies because it's in the dependency tree already